src/pybind/mgr/cephadm/inventory.py

```diff
@@ -843,6 +847,9 @@ def invalidate_host_networks(self, host):
     def distribute_new_registry_login_info(self) -> None:
         self.registry_login_queue = set(self.mgr.inventory.keys())
 
+    def distribute_custom_logrotate_config(self) -> None:
+        self.custom_logrotate_write_queue = set(self.mgr.inventory.keys())
```

Comment:
Is it possible that this line will execute with something already in the queue (set)? We would lose the previous contents.

Reply:
It is absolutely possible, but that old list getting replaced with the set of all hosts is actually fine. Being in this list is just a marker that we want to write out a logrotate config to that host. If you have host A and B in the list, then replace it with host A, B, and C because the logrotate config changed it's okay. We still just need to visit A, B, and C. It doesn't matter at that point that A and B didn't get the previous logrotate config.

Reply:
ok